src/unistd.rs

```diff
@@ -3569,13 +3568,29 @@ impl Group {
     ///
     /// # Examples
```

Comment:
BTW, I think we should move this new test to `tests/test_unistd.rs` since we don't want 2 duplicate examples

Reply:
done